app/shared/recurring-reservation-controls/RecurringReservationControls.js

```diff
@@ -60,10 +60,11 @@ function RecurringReservationControls({
           </Col>
         )}
         {frequency !== '' && (
-          <Col sm={4} xs={12}>
+          <Col sm={5} xs={12}>
             <FormGroup controlId="LastTimeGroup">
               <ControlLabel>{t('RecurringReservationControls.lastTimeLabel')}</ControlLabel>
               <DatePicker
+                className="date-picker"
```

Comment:
Please remove, className date-picker already injected in container
Check app/shared/date-picker/DatePicker.js